docs/guides/using-theme-overrides.md

```diff
@@ -0,0 +1,418 @@
+---
+title: Using theme overrides
+category: Guides
+order: 4
+---
+
+## Using theme overrides
+
```

Comment:
looks nice! I'd add some text to the "theming" page too that links here. I'd also add here some introductory text what this document is about, something like:

This document details how you can customize the Instructure UI components by tweaking their theme variables. While this gives you a level of flexibility on the look and feel of the components you should be aware of 2 things:
- The default theme variables are tested to have high enough contrast ratios for WCAG conformance, if you are making changes it's you job, that your app stays WCAG conformant.
- You cannot fully customize the look of components, just to a certain degree. This is intentional, Instructure UI is a design system geared towards the Canvas look and feel, not a generic UI component library.

Reply:
I added the info, have a look again please :)